### DISCUSSION TITLE
ISCH (Schmidt Rebound Hardness) group

### DISCUSSION THREAD

**Discussion by richard-sisson** [australiangeomechanics/agsformat#60] [Q&A] [answered]
Hi,

Can you confirm the datatype is correct for AXIS (X type) - Description says: ...(positive numbers downwards and negative numbers upward).

Likewise, is orientation a numeric value?
ORN (X type) - Description says: Orientation of hammer axis with reference to intact rock anisotropy features (e.g. lamination, foliation, schistosity, lineation)

Thanks
Richard

**Comment by SolidRockProgrammer** [comment, marked as answer]
The example is "Perpendicular" and "Perpendicular to lamination", I thing that provides an idea of what the author expects.  If one does use numbers we have defined an origin.
